src/App.tsx

```diff
@@ -0,0 +1,200 @@
+import "./App.css";
+import React from "react";
+import { Accordion, Panel } from "baseui/accordion";
+import { Button, KIND, SHAPE } from "baseui/button";
+import { Card, StyledBody, StyledAction } from "baseui/card";
+import { Checkbox, LABEL_PLACEMENT } from "baseui/checkbox";
+import { FormControl } from "baseui/form-control";
+import { Input } from "baseui/input";
+import { Slider } from "baseui/slider";
+import { useStyletron } from "baseui";
+
+function App() {
+  const [css, theme] = useStyletron();
+
+  const [password, setPassword] = React.useState<string>("");
+  const passwordRef = React.useRef(null);
+
+  const [length, setLength] = React.useState<number[]>([10]);
+  const [capitalAllowed, setCapitalAllowed] = React.useState<boolean>(false);
+  const [numberAllowed, setNumberAllowed] = React.useState<boolean>(false);
+  const [symbolAllowed, setSymbolAllowed] = React.useState<boolean>(false);
+  const [copied, setCopied] = React.useState<boolean>(false);
+
+  // GENERATE PASSWORD
+  const generatePassword = () => {
+    const passwordLength: number = length[0];
+
+    let stringSet: string = "abcdefghijklmnopqrstuvwxyz";
+    if (capitalAllowed) stringSet += "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
+    if (numberAllowed) stringSet += "0123456789";
+    if (symbolAllowed) stringSet += `!"#$%&\:;<=>?@'({|}~)*+,-[\\]^_./`;
+
+    let tempPassword: string = "";
+    for (let i = 0; i < passwordLength; i++) {
+      const charIndex: number = Math.floor(Math.random() * stringSet.length);
+      tempPassword += stringSet[charIndex];
+    }
+    setPassword(tempPassword);
+    setCopied(false);
+  };
+
+  React.useEffect(() => {
+    generatePassword();
+  }, [capitalAllowed, numberAllowed, symbolAllowed, length]);
+
+  return (
+    <>
+      <Card
+        overrides={{
+          Root: {
+            style: {
+              maxWidth: "400px",
+              position: "absolute",
+              top: "20px",
+              right: "50%",
+              transform: "translate(50%, 0)",
+              width: "95%",
+              backgroundColor: "#121212",
+              border: "none",
+            },
+          },
```

Comment:
instead of specifying all these styles inline, can you use either CSS/SCSS stylesheets or a separate styles.ts file where all these objects can be put together.

It doesn't look clean to have the HTML + CSS thrown together